charts/event-handler/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for email.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# replicaCount: 1
+
+image:
+  repository: 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-email
```

Comment:
```suggestion
  repository: 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-event-handler
```

Reply:
Yeah, I seemed to have missed some of those here